### ISSUE TITLE
Document missing tags

### ISSUE THREAD

**Issue by MustafaJafar** [ynput/ayon-documentation#311]
### Is there an existing issue for this?

- [x] I have searched the existing issues and added correct labels.

### Please describe the feature you have in mind and explain what the current shortcomings are?

Tags table in  [addon_core_settings#tags](https://ayon.ynput.io/docs/addon_core_settings#tags) is missing some built in tags like:

- `passing`
- `thumbnail`
- `need_thumbnail`
- `toScanline`
- `publish_on_farm`
- `thumb-nuke`

### Suggested implementation?

.

**Comment by MustafaJafar** [comment]
Tagging @jakubjezek001 and @robin-ynput for visibility. 
I made this issue because I found `passing` tag used in the example CSV file (which can be confing).

**Comment by jakubjezek001** [comment]
> Tagging [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/40640033?s=64&amp;u=72e133b50d74675ce5dc8c0d9e04d519f6df77c5&amp;v=4">@jakubjezek001](https://github.com/jakubjezek001?rgh-link-date=2024-12-03T19%3A15%3A35.000Z) and [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/robin-ynput?size=32">@robin-ynput](https://github.com/robin-ynput?rgh-link-date=2024-12-03T19%3A15%3A35.000Z) for visibility. I made this issue because I found `passing` tag used in the example CSV file (which can be confing).

yeah `passing` is good to be documented but it is more CSV related - as you had mentioned
